psi4/driver/procrouting/scf_proc/scf_iterator.py

```diff
@@ -339,6 +323,34 @@ def scf_iterate(self, e_conv=None, d_conv=None):
 
         self.set_energies("Total Energy", SCFE)
         core.set_variable("SCF ITERATION ENERGY", SCFE)
+
+        return SCFE
+
+    # Single-shot calculation
+    if core.get_option('SCF', 'MAXITER') == 0:
+        form_fock()
```

Comment:
I think perhaps a print_out here may be a good idea, perhaps something like " ==> Non-SCF calculation <==" or something more appropriate. Just so it's super obvious what's happening.